zim-tools does not compile anymore

```
[8/45] Compiling C++ object 'src/25a6634@@zimsearch@exe/zimsearch.cpp.o'.
FAILED: src/25a6634@@zimsearch@exe/zimsearch.cpp.o 
ccache c++ -Isrc/25a6634@@zimsearch@exe -Isrc -I../src -I/usr/local/include -fdiagnostics-color=always -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wnon-virtual-dtor -Werror -std=c++11 -g -Werror -Wall '-DVERSION="2.2.0"' -MD -MQ 'src/25a6634@@zimsearch@exe/zimsearch.cpp.o' -MF 'src/25a6634@@zimsearch@exe/zimsearch.cpp.o.d' -o 'src/25a6634@@zimsearch@exe/zimsearch.cpp.o' -c ../src/zimsearch.cpp
../src/zimsearch.cpp: In function ‘int main(int, char**)’:
../src/zimsearch.cpp:68:28: error: no matching function for call to ‘zim::Query::setQuery(std::string&, bool)’
   68 |     query.setQuery(s, false);
      |                            ^
In file included from ../src/zimsearch.cpp:21:
/usr/local/include/zim/search.h:129:12: note: candidate: ‘zim::Query& zim::Query::setQuery(const string&)’
  129 |     Query& setQuery(const std::string& query);
      |            ^~~~~~~~
/usr/local/include/zim/search.h:129:12: note:   candidate expects 1 argument, 2 provided
[17/45] Compiling C++ object 'src/zimcheck/09b87c9@@zimcheck@exe/checks.cpp.o'.
ninja: build stopped: subcommand failed.
```

Against latest dev libzim

--- a/src/zimsearch.cpp
+++ b/src/zimsearch.cpp
@@ -65,7 +65,7 @@ int main(int argc, char* argv[])
     zim::Archive zimarchive(argv[1]);
     zim::Searcher searcher(zimarchive);
     zim::Query query;
-    query.setQuery(s, false);
+    query.setQuery(s);
     auto search = searcher.search(query);
     printSearchResults(search);
   }

